cli/run_util.go

```diff
@@ -847,6 +848,31 @@ func activateAndRunSteps(
 	return buildRunResults
 }
 
+func logStepStarted(stepInfo stepmanModels.StepInfoModel, step stepmanModels.StepModel, idx int, stepExcutionId string, stepStartTime time.Time) {
+	idVersion := ""
+	if stepInfo.Step.Title != nil && *stepInfo.Step.Title != "" {
+		idVersion = *stepInfo.Step.Title
+	}
+
+	title := ""
+	if step.Title != nil {
+		title = *step.Title
+	}
+
+	params := log.StepStartedParams{
+		ExecutionId: stepExcutionId,
+		Position:    idx,
+		IdVersion:   idVersion,
```

Comment:
IMO we don't need both idVersion and title.
We should have one filed, which stores the Step Title to present. The value should follow this precedence:
- override Step title (defined in the bitrise.yml)
- original Step title  (defined in the step.yml)
- stepIDVersionComposite

Reply:
That is a great idea. I will update the code.

Reply:
The header title is much more better now.

This was before:
```
+------------------------------------------------------------------------------+
| (0) script                                                                   |
+------------------------------------------------------------------------------+

+------------------------------------------------------------------------------+
| (1) git::https://github.com/bitrise-steplib/bitrise-step-simple-git-clone... |
+------------------------------------------------------------------------------+

+------------------------------------------------------------------------------+
| (2) certificate-and-profile-installer@1                                      |
+------------------------------------------------------------------------------+
```

And this is after the change:
```
+------------------------------------------------------------------------------+
| (0) Clear step outputs                                                       |
+------------------------------------------------------------------------------+

+------------------------------------------------------------------------------+
| (1) git::https://github.com/bitrise-steplib/bitrise-step-simple-git-clone... |
+------------------------------------------------------------------------------+

+------------------------------------------------------------------------------+
| (2) Certificate and profile installer                                        |
+------------------------------------------------------------------------------+
```